src/draw.c

```diff
@@ -322,7 +322,7 @@ draw_lineno_custom(struct view *view, struct view_column *column, unsigned int l
 		return false;
 
 	if (lineno == 1 || (lineno % interval) == 0) {
-		static char fmt[] = "%ld";
+		static char fmt[] = "%#u";
```

Comment:
I'm reasonably sure `u` can't use `#`. GCC even warns about it.

Reply:
Actually # is just a placeholder, it is replaced with a digit one line below. I'll use %3u for clarity.